tests/helpers/supabaseMockFactory.js

```
@@ -0,0 +1,215 @@
+    neq: jest.fn(() => builder),
+    gt: jest.fn(() => builder),
+    gte: jest.fn(() => builder),
+    lt: jest.fn(() => builder),
+    lte: jest.fn(() => builder),
+    in: jest.fn(() => builder),
+    is: jest.fn(() => builder),
+    order: jest.fn(() => builder),
+    limit: jest.fn(() => builder),
+    single: jest.fn(() => Promise.resolve(resolveValue)),
+    maybeSingle: jest.fn(() => Promise.resolve(resolveValue)),
+    then: (resolve) => resolve(resolveValue)
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)